addons/web/static/src/views/fields/field.js

```diff
@@ -115,6 +115,9 @@ export class Field extends Component {
             classNames[getClassNameFromDecoration(decoName)] = value;
         }
 
+        if (fieldInfo.viewType === "calendar") {
+            classNames['w-100'] = true;
+        }
```

Comment:
why are we doing this here ?

Reply:
hello, @pkyriakou 

I added width to the field here because I couldn't find anything in `calendar_common_popover.xml` within the `<field>` tag to apply width. We need to apply width to the field because it will work for all fields in the calendar popover.

Reply:
Uhm sure :shrug: seems like a strange solution but it can be checked also at the final review. 
Thanks for the explanation :raised_hands:

Reply:
I agree with @pkyriakou it is really strange.

I think you can give `class` property to Field component as we did in the timer header. 
https://github.com/odoo/enterprise/blob/7c8e15829fd86e96cfa3851549ba933ba302dfd0/timesheet_grid/static/src/components/timesheet_timer_header/timesheet_timer_header.xml#L49